src/services/database.ts

```diff
@@ -93,3 +102,28 @@ const CriteriaIndexMap: Record<Exclude<SearchCriteria, SearchCriteria.ALL>, Tabl
   vin: 'VinIndex',
   trailerId: 'TrailerIdIndex',
 };
+export const postTechRecord = async (request: any) => {
+  if (process.env.AWS_SAM_LOCAL) {
```

Comment:
Why is this needed? I thought we could interact locally anyway?

Reply:
Yeah get it for the test number service, but I just think the database method should still be able to run? I know Guillaume put a lot of work into making the local dynamo work

Reply:
I agree, I don't see why the writing of the record would have any issues locally. It's able to pull data, don't see why it wouldn't be able to write data. 

You can "mock" out things like testNumber service when running locally, but this is core functionality, so it should work locally.

Reply:
Are you referring to doing integration tests to test it? So that it will actually hit the docker dynamo?

Reply:
I'm just referring to being able to run it locally. Let's say I run the lambda locally, get postman out and I want to check it's saving the record to the local version of the database. That's what I'm talking about

Reply:
You can run it locally and hit it in postman and it'll be saved in the local dynamoDB?